### ISSUE TITLE
When there are 3 or more local tasks on user/%/*, navigating back to the default local task results in an empty content area

### ISSUE THREAD

**Issue by signalpoint** [signalpoint/DrupalGap#350]
```
user/%/view
user/%/edit
user/%/articles (custom local task)
```

If you navigate to the user's profile, then navigate to any local task (edit, articles), then try to navigate back to the view default local task, the page renders, but the content area is empty. This only seems to happen when looking at your own profile (maybe just limited to use no. 1, but probably not). It's possible to navigate to other user profiles, and switch between all the local tasks without issue.

**Comment by chrisschaub** [comment]
I was just working on this. Interesting, I have an authenticated menu in settings.js that has a a path of "user" ... which seems to work the first time. I click "edit" and then if I click "view", boom, empty view. But if I click edit, save the profile then it sends me to view which is populated.

Maybe related, but I noticed that Drupal.user.uid is 0 in settings.js even though I am logged in.

**Comment by signalpoint** [comment]
@chrisschaub Indeed, this is an odd issue. It just came up recently... so it happened somewhere in here (probably):
- https://github.com/signalpoint/DrupalGap/commits/7.x-1.x/src/includes/menu.inc.js
- https://github.com/signalpoint/DrupalGap/commits/7.x-1.x/src/modules/menu/menu.js

The Drupal.user object will (unfortunately) never be populated in settings.js, because the Drupal site context isn't available until after we load the settings.js file.

**Comment by signalpoint** [comment]
Update, this also happens even with the two default tabs, "View" and "Edit".

When you first navigate to the profile to view it, the page gets this id:

```
user_1_view
```

Then if you navigate to the edit page that page gets this id:

```
user_1_edit
```

But when you navigate back to the "View" tab a new page is generated with this id:

```
user_1
```

**Comment by max1020** [comment]
I am having the same issue - is there a way to get around this? I do not necessarily need the view tab of the user profile. a single menu link to the edit profile settings would work. The menu has "user" as path - how can i set a path to the "edit tab"?

**Comment by signalpoint** [comment]
@max1020 These days, I usually just remove the `primary_local_tasks` from the `settings.js` file. To build dynamic menus, see the docs here: http://docs.drupalgap.org/7/Menus/Menus_with_Dynamic_Links

**Comment by max1020** [comment]
thanks for the tipp. I have removed the primary_local_tasks but I cant get it to work. The menu block shows up, but when I click and navigate 2 or three times it stays on the same page. When I click the back button i get this in the console:

```
 WARNING: theme_() does not exist. Called by: drupalgap_render_page().
2016-01-08 11:01:47.135 paypal3drupal[1310:44996] arg - TypeError: undefined is not an object   (evaluating 'arguments[1].split')
```

I have also tried disabling the User Menu Authenticated menu and corresponding block  (user_menu_authenticated) in settings.js

What does the attribute (and the fq_user_menu in particular) mean in the my_module_block_view function:

```
attributes: {
          id: drupalgap_get_page_id() + '_fq_user_menu'
        },
```

**Comment by signalpoint** [comment]
@max1020 Thanks for pointing that out, that's actually a mistake in the docs. It should be `_my_module_user_menu`, I've updated the docs.

**Comment by max1020** [comment]
Hey Tyler! Thanks. I cant get it to work on ios? The corrected code works perfectly fine as webapp, but when I try to get it work on the simulator and real device the same error occurs and once I accessed the edit tab, I cant navigate anywhere else. The app starts and worksflawlessly in ios with the exception of this. any idea? do i need something else to make this work on ios?
